python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -1748,6 +1749,7 @@ def my_dbt_op(dbt: DbtCliResource):
         if self.target:
             profile_args += ["--target", self.target]
 
+        cli_args = args
         args = [
```

Comment:
nit: rather than introducing `cli_args`, let's just change the name of this reassignment to something like `full_dbt_args` or something